HomeWork_w02_D01/AboutPage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>About Me</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="ab.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <hr size ="10" color ="black" width="900" >
+
+    <nav>
+        <ul>
+            <li><a href="AboutPage.html">About Me | </a></li>
+            <li><a href="portfilo.html">Portfilo</a></li>
+        </ul>
+    </nav>
+
+
+    <h1>
+        Mohammed Jameel
+    </h1>
+
+    <hr size ="5"  color="black" width="850" >
+
+    <img class="logo" src="/Users/mo7amed178/WDI/lesson-week_02-day_01-css/HomeWork_w02_D01/Images/person.jpg">
```

Comment:
So this would only work on your computer since the path to the file is specific to your computer, to solve this; you can use [relative paths](https://www.w3schools.com/html/html_filepaths.asp) so it works for every computer.